src/metabase/api/geojson.clj

```diff
@@ -83,9 +83,11 @@
   :default {}
   :getter  (fn [] (merge (setting/get-json :custom-geojson) builtin-geojson))
   :setter  (fn [new-value]
-             (when new-value
-               (validate-geojson new-value))
-             (setting/set-json! :custom-geojson new-value))
+             ;; remove the built-in keys you can't override them and we don't want those to be subject to validation.
+             (let [new-value (not-empty (reduce dissoc new-value (keys builtin-geojson)))]
+               (when new-value
+                 (validate-geojson new-value))
+               (setting/set-json! :custom-geojson new-value)))
```

Comment:
This was the only thing that broke, because the built-in GeoJSON entries that we got out failed validation going back in, because they're local resources. We don't support overriding those anyway, since we `merge` `builtin-geojson` last in the `:getter` above. So I just tweaked the setter to ignore the built-in values